main.go

```diff
@@ -97,15 +105,31 @@ func main() {
 	service.Use(middleware.ErrorHandler(service, true))
 	service.Use(middleware.Recover())
 
-	publicKey, err := token.ParseRSAPublicKeyFromPEM([]byte(RSAPublicKey))
+	publicKey, err := token.ParsePublicKey(token.RSAPublicKey)
+	if err != nil {
+		panic(err)
+	}
+
+	privateKey, err := token.ParsePrivateKey(token.RSAPrivateKey)
 	if err != nil {
 		panic(err)
 	}
+
+	tokenManager := token.NewManager(publicKey, privateKey)
 	app.UseJWTMiddleware(service, jwt.New(publicKey, nil, app.NewJWTSecurity()))
 
 	// Mount "login" controller
-	loginCtrl := NewLoginController(service)
+	oauth := &oauth2.Config{
+		ClientID:     "875da0d2113ba0a6951d",
+		ClientSecret: "2fe6736e90a9283036a37059d75ac0c82f4f5288",
```

Comment:
Note: To be handled during deployment as part of https://github.com/almighty/almighty-core/issues/326